utils/ICutils.py

```diff
@@ -64,8 +65,8 @@ def load_pybasic_data(
     # This for loop will run through the paths to the images for the specified channel and load in the images to be used to illumination correction
     # This code was sampled from the `load_data` function in PyBaSiC
     for i, image_file in enumerate(image_files):
-        if verbosity and (i % 10 == 0):
-            print(i, "/", len(image_files))
+        # if verbosity and (i % 10 == 0):
+        #     print(i, "/", len(image_files))
```

Comment:
Are these still needed? If not, consider removing. This may also relate to the function parameter overall (are there any other usecases?).

Reply:
PyBaSiC isn't needed. I will be moving this into a hidden folder so that if I want to go back in the future and use it, I will have the older code.